src/src/PluginStructs/P092_data_struct.cpp

```diff
@@ -434,7 +434,7 @@ boolean DLBus::CheckCRC(uint8_t IdxCRC) {
   return false;
 }
 
-// sensor types
+/ sensor types
```

Comment:
Huh?
I don't get this.
Change from a comment to something that will not compile.

Reply:
My mistake, changed in the next commit.